src/tracker.js

```diff
@@ -0,0 +1,18 @@
+const createTracker = page => {
+  const requests = new Set();
```

Comment:
Why a set? How is a request hashed to be unique? 
Also what happens if you have something like:
```javascript
$(function() {
  window.setInterval(function() {
    fetch('/api/check-for-new-content').then(...)
  }, 1000)
}
```
Or if there's a repeated `<img src="same.png">` and the URL it resolves to doesn't have cache-control.

Reply:
Nevermind this. A set is fine. It keeps it simple. We can revisit this later.